guidelines.md

```diff
@@ -54,23 +63,20 @@ In case of any error, the returned JSON object describes the error and has the f
 | `Message` | string | required | Description of the error. |
 | `Details` | string | optional | Additional details about the error \(request, headers, server stack trace, inner exceptions etc.\). Only available on development environment. |
 
-Some errors may also contain additional information relevant to the error on top of this two properties. But that depends on the operation and is specifically described in the operation documentation.
+Some errors may also contain additional information relevant to the error on top of these two properties. That depends on the operation and is specifically described in the operation documentation.
 
 ## Authentication
 
-Authentication and tokens that have to be used depending on environment.
+**The demo environments listed below are completely public and NO REAL DATA should be used for any reason. Failure to comply with these guidelines can result in immediate suspension of the connection or denial of certification.**
```

Comment:
I think it doesn't belong here. There should be only credentials in this section. I would create new section above, called e.g Security, Security policy, ...and I would place it there.

Reply:
Or place it after the demo entironments

### Demo environments

#### Security policy

Reply:
Went with option B. Also for consistency moved initial line 45 to Security policy under Production environment